requirements.txt

```diff
@@ -6,4 +6,5 @@ terminaltables
 pycocotools
 torch>=1.1
 torchvision
-imagecorruptions
\ No newline at end of file
+imagecorruptions
+albumentations==0.3.2
```

Comment:
We may just keep track of the latest albumentations.

Reply:
Changed to >=